frontend/www/js/omegaup/components/Autocomplete.vue

```diff
@@ -39,8 +45,17 @@ export default class Autocomplete extends Vue {
   }
 
   @Watch('value')
-  onPropertyChanged(newValue: string, oldValue: string) {
+  onValueChanged(newValue: string, oldValue: string) {
+    const alias = this.input.getAttribute('data-alias');
+    if (alias !== null) {
+      this.alias = alias;
+    }
```

Comment:
esto se puede reemplazar por

```suggestion
    if (alias !== null) {
      this.$emit('update:value', value);
    }
```

Reply:
Listo